src/lib/yauthclient/main_dialog.rb

```diff
@@ -109,7 +109,7 @@ def render_section_tree
             # For the currently selection config section, render customised parameters and values in a table.
             def render_section_conf
                 Yast::UI.ReplaceWidget(Id(:section_conf), VBox(
-                    Left(Label(Opt(:boldFont), _("Customisation - %s" % UIData.instance.get_curr_section))),
+                    Left(Label(Opt(:boldFont), _("Customisation - %s") % UIData.instance.get_curr_section)),
```

Comment:
Well, actually translators had already managed to translate the text. I'll do my best trying to add useful a comment, anyways.